From d7e6b921cd34f7ad8fc4d05065c75d13ba330896 Mon Sep 17 00:00:00 2001
From: Tomas Mraz <tmraz@fedoraproject.org>
Date: Fri, 17 Aug 2012 12:46:40 +0000
Subject: Add missing $(DESTDIR) when making directories on install.

modules/pam_namespace/Makefile.am: Add missing $(DESTDIR) when making
$(namespaceddir) on install.
modules/pam_sepermit/Makefile.am: Add missing $(DESTDIR) when making
$(sepermitlockdir) on install.
---
diff --git a/modules/pam_namespace/Makefile.am b/modules/pam_namespace/Makefile.am
index a28f196..ebb00f3 100644
--- a/modules/pam_namespace/Makefile.am
+++ b/modules/pam_namespace/Makefile.am
@@ -40,7 +40,7 @@ if HAVE_UNSHARE
   secureconf_SCRIPTS = namespace.init
 
 install-data-local:
-	mkdir -p $(namespaceddir)
+	mkdir -p $(DESTDIR)$(namespaceddir)
 endif
 
 
diff --git a/modules/pam_sepermit/Makefile.am b/modules/pam_sepermit/Makefile.am
index cfc5594..bc82275 100644
--- a/modules/pam_sepermit/Makefile.am
+++ b/modules/pam_sepermit/Makefile.am
@@ -35,7 +35,7 @@ if HAVE_LIBSELINUX
   securelib_LTLIBRARIES = pam_sepermit.la
 
 install-data-local:
-	mkdir -p $(sepermitlockdir)
+	mkdir -p $(DESTDIR)$(sepermitlockdir)
 endif
 if ENABLE_REGENERATE_MAN
 noinst_DATA = README pam_sepermit.8 sepermit.conf.5
--
cgit v0.9.1
